### PR TITLE
Add ssh connection retry function

### PR DESCRIPTION
**Pull request by yuhiwa**
- Sometimes, ssh connection failed in my environment.
  So I'd like to add ssh retry connection function in case of network error.
- add 3 parameters for retry ssh connection
  
  - initial_retry_wait
  - max_retry_wait
  - max_retry_limit

### CODE REVIEW

File: src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java [hunk 1]
```diff
@@ -61,6 +64,9 @@ public SshOperator(OperatorContext context)
         }
 
         private final int defaultCommandTimeout = 60;
+        private final int initialRetryWait = 500;
+        private final int maxRetryWait = 2000;
+        private final int maxRetryLimit = 3;
```
Comment (hiroyuki-sato) [inline src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java:70]: (1) What do you think to add `default_` prefix?
Like `defaultInitialRetryWait` or `DEFAULT_INITIAL_RETRY_WAIT`
Ref: https://github.com/treasure-data/digdag/pull/994/files#diff-84a9dc43263ac1f78fa3b2aca8234f6fR160

(2) `private static final` is better? (It should change `defaultCommandTimeout` by the way ^^;; )

**Comment by hiroyuki-sato** [comment]
Hello, @yuhiwa 

Thank you for creating this PR.

I modified the Digdag code lately. And I felt that the default parameter needs `DEFAULT_` prefix and `private static final`.
If you feel that is better, please modify your code, Later I'll change `defaultCommandTimeout` too.

File: src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java (hunk 1 above)
Comment (yuhiwa) [reply src/main/java/com/github/hiroyuki_sato/digdag/plugin/SshOperatorFactory.java:70]: (1)
Just I forgot default prefix.

(2)
yes, private static final is better, and I'll change defaultCommandTimeout, too.

**Comment by hiroyuki-sato** [comment]
LGTM 👍

I'll test this PR in my environment later. Please wait.
If you want to use a new version ASAP, I'll merge it and release a new version right now.

**Comment by hiroyuki-sato** [comment]
LGTM. merging.

**Pull request merged**